node/helper-module/index.js

```diff
@@ -0,0 +1,26 @@
+const iopipe = require('@iopipe/iopipe')();
+
+// modularize the tracing functionality for use in other modules outside of "main"
+const { setContext, label, log } = require('./iopipe-helper');
+
+// normal lambda work
+const { typeOne, typeTwo } = require('./dog');
+
+exports.handler = iopipe(async (event, context) => {
+  // important! reset context for each invocation
+  setContext(context);
+
+  // do your work as normal
+  const pongo = await typeOne();
+  const perdita = await typeTwo();
+
+  // add IOpipe labels to the invocation, 1 label per dog breed
+  const breeds = [pongo, perdita].map(url => url.match(/\/breeds\/(.*)\//)[1]);
+  breeds.forEach(label);
+
+  // add IOpipe custom metrics for each url
+  log('pongo-url', pongo);
```

Comment:
@coreylight Don't we want to encourage use of `metric()` since `log()` is being deprecated?

Reply:
@kolanos yup you're right, I'll change that. Thanks!